pkgs/tools/networking/doggo/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  runCommand,
+  installShellFiles,
+}: let
+  buildDate = "2022-08-15T21:36:49";
+  buildCommit = "2cf9e7b";
+in
+  with lib;
+    buildGoModule rec {
+      pname = "doggo";
+      version = "0.5.4";
+      vendorSha256 = "sha256-pyzu89HDFrMQqYJZC2vdqzOc6PiAbqhaTgYakmN0qj8=";
```

Comment:
vendorSha256 is usually goes after the src field, in between sha256 and nativeBuildInputs. Just a stylistic convention though.